cg/meta/workflow/rnafusion.py

```diff
@@ -0,0 +1,315 @@
+        LOG.info("Samplesheet written")
+
+    def run_analysis(
+        self,
+        case_id: str,
+        log: str,
+        work_dir: str,
+        resume: bool,
+        profile: str,
+        with_tower: bool,
+        stub: bool,
+        input: str,
+        outdir: str,
+        genomes_base: str,
+        trim: bool,
+        fusioninspector_filter: bool,
+        all: bool,
+        pizzly: bool,
+        squid: bool,
+        starfusion: bool,
+        fusioncatcher: bool,
+        arriba: bool,
+        dry_run: bool = False,
+    ) -> None:
+        """Execute RNAFUSION run analysis with given options"""
+
+        options = self.__build_command_str(
+            self.get_verified_arguments(
+                case_id=case_id,
+                work_dir=work_dir,
+                resume=resume,
+                profile=profile,
+                with_tower=with_tower,
+                stub=stub,
+                input=input,
+                outdir=outdir,
+                genomes_base=genomes_base,
+                trim=trim,
+                fusioninspector_filter=fusioninspector_filter,
+                all=all,
+                pizzly=pizzly,
+                squid=squid,
+                starfusion=starfusion,
+                fusioncatcher=fusioncatcher,
+                arriba=arriba,
+            )
+        )
+        nextflow_options = self.__build_command_str(
+            self.get_verified_arguments_nextflow(
+                case_id=case_id,
+                log=log,
+            )
+        )
+        command = ["run", self.nfcore_pipeline_path]
+        parameters = (
+            nextflow_options
+            + ["-bg", "-q"]
+            + command
+            + options
+            + [
+                " > "
+                + str(self.get_case_path(case_id))
+                + "/"
+                + case_id
+                + "-stdout.log 2> "
+                + str(self.get_case_path(case_id))
+                + "/"
+                + case_id
+                + "-stdout.err  < /dev/null & "
+            ]
+        )
+        self.process.run_command(parameters=parameters, dry_run=dry_run)
+
+    def get_deliverables_file_path(self, case_id: str) -> Path:
+        """Returns a path where the rnafusion deliverables file for the case_id should be located.
+
+        (Optional) Checks if deliverables file exists
+        """
+        return Path(self.get_case_path(case_id), case_id + "_deliverables.yaml")
+
+    def get_template_deliverables_file(self, rnafusion_bundle_template: Path) -> pd.DataFrame:
+        """Read deliverables file template and output to dataframe format"""
+        return pd.read_csv(rnafusion_bundle_template)
+
+    def edit_template_deliverables_file(
+        self, case_id: str, deliverables_template: pd.DataFrame
+    ) -> pd.DataFrame:
+        """Replace PATHTOCASE and CASEID from template deliverables file to corresponding strings, add path_index column"""
```

Comment:
```suggestion
        """Parse template deliverables file."""
```
Apparently, a docstring should have the first character in upper case and end with a dot (I had to look it up). I usually have removed dots from docstrings, but I think we should add them back in again if this is the python standard

Reply:
1f0fc2ec8f280437e0cdff14b44ebcf8729546e8 added dots to rnafusion files but not the rest of the code